tests/testthat/test-post_bug.R

```diff
@@ -0,0 +1,18 @@
+cli::test_that_cli(configs = "plain", "post_bug() works", {
+    skip_if_not(interactive())
+    vcr::use_cassette("post_bug_works", {
+        pg <- post_bug()
+    })
+    expect_snapshot(
+        expect_type(pg, "list")
+    )
+})
+
+
+# cli::test_that_cli(configs = "plain", "post_bug() fails", {
+#     skip_if_not(interactive())
+#     pg1 <- post_bug()
+#     expect_snapshot(
+#         expect_error(pg1, "Do it and then return please.")
```

Comment:
If you want to test an error or message you need to use `expect_error(post_bug(), "Do it an then return please")`. Although by being on a snapshot I think you do not need to add any other expect_error function if the function raising the error (`post_bug` is inside it: `expect_snapshot(post_bug())`